docs/source/llms/dspy/notebooks/dspy_quickstart.ipynb

```diff
@@ -0,0 +1,1265 @@
+      "Incorrect prediction:    finance\n",
+      "Incorrect prediction:    financial_report\n",
+      "Incorrect prediction:    merger_news\n",
+      "Incorrect prediction:    Agriculture\n",
+      "Incorrect prediction:    trade relations\n",
+      "Incorrect prediction:    Labor Strike\n"
+     ]
+    }
+   ],
+   "source": [
+    "for uncompiled_residual, uncompiled_prediction in zip(uncompiled_residuals, uncompiled_predictions):\n",
+    "    is_correct = \"Correct\" if bool(uncompiled_residual) else \"Incorrect\"\n",
+    "    prediction = uncompiled_prediction.label\n",
+    "    print(f\"{is_correct} prediction: {' ' * (12 - len(is_correct))}{prediction}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "e93c1b68-a95e-436e-9c0d-54b2b3e34f6f",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Correct prediction:      interest\n",
+      "Incorrect prediction:    politics\n",
+      "Correct prediction:      money-fx\n",
+      "Correct prediction:      earn\n",
+      "Correct prediction:      acq\n",
+      "Correct prediction:      grain\n",
+      "Correct prediction:      trade\n",
+      "Incorrect prediction:    strike\n"
+     ]
+    }
+   ],
+   "source": [
+    "for compiled_residual, compiled_prediction in zip(compiled_residuals, compiled_predictions):\n",
+    "    is_correct = \"Correct\" if bool(compiled_residual) else \"Incorrect\"\n",
+    "    prediction = compiled_prediction.label\n",
+    "    print(f\"{is_correct} prediction: {' ' * (12 - len(is_correct))}{prediction}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "73351663-ed84-4e29-8bf5-61fbf94da937",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# 3 - Load and Load out Model with MLflow\n",
+    "\n",
+    "Now that we have a compiled model with higher classification accuracy, let's leverage MLflow to log this model and load it for inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading artifacts: 100%|██████████| 7/7 [00:00<00:00, 2039.04it/s] \n"
+     ]
+    }
+   ],
+   "source": [
+    "import mlflow\n",
+    "from mlflow.models import ModelSignature\n",
+    "from mlflow.types.schema import ColSpec, Schema\n",
+    "\n",
+    "input_schema = Schema([ColSpec(\"string\")])\n",
+    "output_schema = Schema([ColSpec(\"string\")])\n",
```

Comment:
same here - let's delete the schema definition, and let mlflow infer from the input_example